backend/tests/pps/test_sch.py

```diff
@@ -13,21 +13,21 @@ def test_total_potency(self):
             wd=180, mainstat=5577, det=2272, crit=3802, dh=1100, speed=2139, ten=380, pie=340)
         test_char = Character(Jobs.SCH, my_stat_spread)
         mypps = SchPps()
-        self.assertEqual(24326.1264, mypps.total_potency_spreadsheet_port(test_char, 0.12, 4, 0))
+        self.assertAlmostEqual(24326.1264, mypps.total_potency_spreadsheet_port(test_char, 0.12, 4, 0), places=4)
```

Comment:
why 4 digits here instead of 3?

Reply:
it was the only value that had four digits instead of like twelve so i thought there was some significance 

brought it in line with the other values

Reply:
am i losing my mind or do i still see places=4 here

Reply:
fixed for real this time